openedx/core/djangoapps/commerce/utils.py

```diff
@@ -33,8 +33,16 @@ def is_commerce_service_configured():
 
 def ecommerce_api_client(user, session=None):
     """ Returns an E-Commerce API client setup with authentication for the specified user. """
-    claims = {'tracking_context': create_tracking_context(user)}
-    jwt = create_jwt_for_user(user, additional_claims=claims)
+    claims = {
+        'tracking_context': create_tracking_context(user),
+        'user_id': user.id,
```

Comment:
I _think_ the scope should be enough to bring this in.

Reply:
Yep. I grabbed a resulting jwt from a unit test, decoded it, and it did have the user_id. I've updated the file.